homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -133,7 +133,9 @@ class AlarmControlPanelEntity(Entity):
     _attr_changed_by: str | None = None
     _attr_code_arm_required: bool = True
     _attr_code_format: CodeFormat | None = None
-    _attr_supported_features: AlarmControlPanelEntityFeature | int = 0
+    _attr_supported_features: AlarmControlPanelEntityFeature = (
+        AlarmControlPanelEntityFeature(0)
```

Comment:
I like this more than #82312.

Mainly because #82312 introduces `AlarmControlPanelEntityFeature.NONE`, and `NONE` isn't a feature. So I think this solution is more clean 👍